templates/css/preview.css

```diff
@@ -10,11 +10,20 @@ p {
 article {
   padding: 1px 0;
   padding-bottom: 16px;
-  margin: 0 auto;
-  max-width: 800px;
   background: #fff;
   box-shadow: 0 2px 2px 0 rgba(0,0,0,.14),0 3px 1px -2px rgba(0,0,0,.2),0 1px 5px 0 rgba(0,0,0,.12);
   display: block;
+  margin: 0 auto;
+  max-width: 800px;
+}
+
+#experimental-mode {
+  max-width: 800px;
+  margin: 16px auto;
```

Comment:
I've set the left/right margin to auto in order to align it with the article below.